components/calendly_v2/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@pipedream/calendly_v2",
+  "version": "1.0.0",
+  "description": "Pipedream Calendly_v2 Components",
```

Comment:
The description can be written without the underscore to enhance readability

Reply:
+1